### PR TITLE
Database

### PR DESCRIPTION
**Pull request by amanshawar**

### CODE REVIEW

File: src/database/models/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+serviceDetails.belongsTo(user, {
+  onDelete: 'CASCADE', forignKey: 'user_id', targetKey: 'id'
+});
+
+serviceDetails.belongsTo(services, {
+  onDelete: 'CASCADE', forignKey: 'service_id', targetKey: 'id'
+});
+
+extraServices.belongsTo(serviceDetails, {
+  onDelete: 'CASCADE', forignKey: 'serviceDetails_id', targetKey: 'id'
```
Comment (Samaamro20) [inline src/database/models/index.js]: serviceDetails_id    * Wrong capital var name

Comment (Samaamro20) [inline src/database/models/index.js]: serviceDetails_id * Wrong capital var name

File: src/database/models/extraServices.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const Sequelize = require('sequelize');
+const seq = require('../config/dbConnection');
+
+const extraServices = seq.define('extraServices',{
```
[the remark this answers is not in the document]
Comment (amanshawar) [reply src/database/models/extraServices.js]: done

File: src/database/models/user.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  password: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  email: {
+    type: Sequelize.STRING,
+    allowNull: false,
```
[the remark this answers is not in the document]
Comment (amanshawar) [reply src/database/models/user.js]: done

File: src/database/config/dbConnection.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const Sequelize = require('sequelize');
+
+module.exports = new Sequelize('parties', 'admin', 'admin', {
```
[the remark this answers is not in the document]
Comment (amanshawar) [reply src/database/config/dbConnection.js]: done

File: src/database/models/index.js (hunk 1 above)
Comment (amanshawar) [reply src/database/models/index.js]: done

**Pull request merged**